install.js

```diff
@@ -24,7 +24,7 @@ var paths = {
 if (!paths[platform]) throw new Error('Unknown platform: ' + platform)
 
 // downloads if not cached
-download({version: version}, extractFile)
+download({version: version, arch: process.env.npm_config_arch}, extractFile)
```

Comment:
The environment variable? `npm_config_arch` is automatically set when using the `--arch` flag with `npm`, which is why I used it (see docs I added).